File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        numFreqMap = {}
        for num in nums:
            numFreqMap[num] = numFreqMap.get(num, 0) + 1
        sortedPairs = sorted(numFreqMap.items(), key=lambda x: x[1], reverse = True)
        res = []
        for i in range(k):
            res.append(sortedPairs[i][0])
        return res

        

File: Data Structures & Algorithms/jump-game/submission-0.cpp
class Solution {
public:
    bool canJump(vector<int>& nums, int idx = 0) {
        if (idx == nums.size() - 1) return true;
        for (int i = 1; i <= nums[idx]; i++) {
            if (canJump(nums, i + idx)) return true;
        }
        return false;
    }
};
